PyTest_POM_PoC/tests/test_home_page.py

```diff
@@ -0,0 +1,13 @@
+from pages.home_page import HomePage
+
+
+def test_page_header(driver, test_data):
+    home_page = HomePage(driver)
+    url = test_data["base_url"]
```

Comment:
U can move (define) it in the `HomePage` instead, bc this is your base URL (Home Page)